### PR TITLE
Issue #129: Add latest folder with build_info.properties

### LINKED ISSUE #129: Add latest folder with build_info.properties
Add latest folder with build_info.properties in download area

### PR DESCRIPTION
**Pull request by josephca**
fixes #129

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -61,15 +61,45 @@ spec:
                     unstash 'deployables'
                     sh '''#!/usr/bin/env bash
 
-                    export sshHost="genie.codewind@projects-storage.eclipse.org"
-                    export deployParentDir="/home/data/httpd/download.eclipse.org/codewind/codewind-vscode"
-
+                    export REPO_NAME="codewind-vscode"
+		    export OUTPUT_NAME="codewind"
+	       	    export OUTPUT_THEIA_NAME="codewind-theia"
+ 		    export OUTPUT_DIR="$WORKSPACE/dev/ant_build/artifacts"
```
Comment (tetchel) [inline Jenkinsfile]: can we name this something other than `ant_build`

Comment (tetchel) [reply Jenkinsfile]: actually it appears to not be used at all

**Comment by tetchel** [comment]
tabbing 
image

**Comment by tetchel** [comment]
I would rather capture all of this into a script but that is optional.

File: Jenkinsfile (hunk 1 above)
Comment (josephca) [reply Jenkinsfile]: Good catch. Fixed.

**Pull request merged**